app/views/docs/configuration.phtml

```diff
@@ -168,6 +172,29 @@
     </tr>
     </thead>
     <tbody>
+    <tr>
+        <td>_APP_DOMAIN</td>
+        <td>
+            Your Appwrite domain address. 
+            When setting a public suffix domain, Appwrite will attempt to issue a valid SSL certificate automatically. 
```

Comment:
We actually don't generate SSL certificates automatically anymore (as of 1.4.0). We plan on bringing this back, but TBD when. For now, they would have to run https://appwrite.io/docs/certificates#manualGeneration

This might mean we need to update https://appwrite.io/docs/certificates too.

@Meldiron, can you please confirm?